pkg/blobinfocache/boltdb/boltdb.go

```diff
@@ -1,4 +1,5 @@
-package blobinfocache
+// Package boldtb implements a BlobInfoCache backed by BoltDB.
```

Comment:
Tests fail with:
```
git-validation -q -run DCO,short-subject,dangling-whitespace
/gopath/src/github.com/containers/image/pkg/blobinfocache/boltdb/boltdb.go:1:1: package comment should be of the form "Package boltdb ..."
```

`s/boldtb/boltdb/` it seems.

Reply:
> `s/boldtb/boltdb/` it seems.

Done with b2f8be9 -> d545a19.